PolyEngine/Engine/Src/Movement/FreeFloatMovementComponent.hpp

```diff
@@ -16,8 +16,8 @@ namespace Poly {
 		float GetAngularVelocity() const { return RotationSpeed; }
 
 	private:
-		float MovementSpeed = 1.0f;
-		float RotationSpeed = 1.0f;
+		float MovementSpeed = 10.0f;
+		float RotationSpeed = 10.0f;
```

Comment:
Why? This is engine wide change for what I assume is movement speed in rendering sandbox. Move this to RenderingSandbox project.